app/src/main/java/fr/tvbarthel/apps/shapi/MainActivity.java

```diff
@@ -1,8 +1,14 @@
 package fr.tvbarthel.apps.shapi;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
 
+/**
+ * Main Activity.
+ * <p>
+ * Never rename or move package.
+ * https://android-developers.googleblog.com/2011/06/things-that-cannot-change.html
```

Comment:
Simply use an alias if you want to replace the activity with the main/launcher intent-filter:

http://blog.danlew.net/2014/01/16/preserve-your-launchers-use-activity-alias/